code/modules/food&drinks/food/snacks_pizza.dm

```diff
@@ -172,6 +175,9 @@
 			tagimg.pixel_y = boxes.len * 3
 			overlays += tagimg
 
+		if(has_c4)
+			var/image/c4img = image('icons/obj/assemblies.dmi', "plastic-explosive2")
```

Comment:
why wouldn't you reference the c4 overlay?
`overlays += obj/item/weapon/c4/image_overlay`